app/views/articles/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<p>
+	<strong>Tittel:</strong>
+	<%= @article.title %>
+</p>
```

Comment:
It's not very common to have the words "Tittel:" and "Tekst:" in front of the content on a blog, is it? 😉 

Also, the title should probably be in a `h1` or `h2` tag.